src/components/templates/PartyMap/components/RoomModalOngoingEvent/RoomModalOngoingEvent.scss

```diff
@@ -26,8 +26,4 @@
     margin-bottom: 10px;
     white-space: pre-wrap;
   }
-
-  .room-entry-button {
-    width: 100%;
-  }
```

Comment:
[note] Moved to `src/components/templates/PartyMap/components/RoomModal/RoomModal.scss` as `&__btn-enter`